browse/scripts/main.py

```diff
@@ -39,18 +39,9 @@
 # topics
 tp = TopicsParser()
 topics = tp.gen_topics("../../database/db.csv")
+topics_file = open("../topics/topics-list.md", "w", encoding="UTF-8")
 for topic in topics:
     f.gen_out("../../database/db.csv", '../topics/{}.md'.format(topic), "topic", "{}".format(topic))
+    topics_file.write("[" + topic + "]" + "(" + "../topics/{}.md".format(topic) + ")" + "<br>")
```

Comment:
instead of a `<br>` I think it would be better to use `\n` (newline characters) to split the line. In this way the topics-list file be easy to edit manually (whenever the need be) in its raw format. Without the newline character, the raw file looks a like a paragraph.